tendrl/alerting/handlers/__init__.py

```diff
@@ -165,7 +165,7 @@ def run(self):
                     new_alert_str = NS.etcd_orm.client.read(
                         '/Messages/events/%s/payload' % new_msg_id,
                         recursive=True
-                    ).leaves[0].value
+                    ).leaves.next().value
```

Comment:
python-etcd returns EtcdResult which contains a private field called _children which was used earlier but later as part of namespace changes PR, in accordance with some comments but actually _children is a private array whereas leaves provides instance of generator which cannot be accessed as an array but needs to be used as.next() So, this change.